public/js/composer.js

```diff
@@ -55,4 +59,37 @@ $(document).ready(function () {
         };
     });
 
+    $(window).on('action:quill.load', function (ev, quill) {
+        require(['quill', 'composer/formatting'], function (Quill, formatting) {
+            // Override the composer-default handler for ns-spoiler and use one for Quill
+            const Delta = Quill.import('delta');
```

Comment:
@julianlam what is the difference between `Quill` and `quill` provided in the load action (line 62)? Does `quill` instance have an `import` method?

Reply:
Quill is the library itself, lowercase quill is the instantiated object 🙂

Yes I agree it's confusing, I have to refresh my memory every time lol